cas-server-integration-infinispan/build.gradle

```diff
@@ -0,0 +1,35 @@
+description = 'Apereo CAS Infinispan Integration'
+dependencies {
+    compile group: 'org.springframework', name: 'spring-context-support', version:springVersion
+    compile group: 'org.springframework', name: 'spring-context', version:springVersion
+    compile(group: 'org.infinispan', name: 'infinispan-core', version:infinispanVersion) {
+        exclude(module: 'guava')
+        exclude(module: 'jboss-logging')
```

Comment:
jboss logging itself is a bridge that can work with slf4j. Since we already include jboss-logging on the classpath via hibernate, we are excluding it from here to remove versioning conflicts.